server/controllers/boards.js

```diff
@@ -0,0 +1,60 @@
+const Board = require('../models/Board');
+const Column = require('../models/Column');
+const User = require('../models/Users');
+const asyncHandler = require('../middlewares/asyncHandler');
+const { ErrorResponse} = require('../utils/errorResponse');
+
+/*Create new board for the new account*/
+exports.initializeFirstBoard = async (userObjectId) => {
+    try {
+        //Create a new board
+        const board = await Board.create({title: 'My board', owner: userObjectId})
+
+        //Create new columns 'in progress' and 'completed'
+        const column1 = await Column.create({title: "In progress", boardId: board._id});
+        const column2 = await Column.create({title: "Completed", boardId: board._id});
+
+        board.columns = [column1._id, column2._id];
+        await board.save();
+
+        const user = await User.findById(userObjectId);
+        user.selectedBoard = board._id;
+        user.boards = [board._id];
+        user.save();
+
+        //Return
+        return true;
+    } catch (e) {
+        return new ErrorResponse(e.message, 500)
```

Comment:
Shouldn't you throw this error instead of returning? Otherwise it will be ignored

Reply:
I changed